libvncclient/rfbproto.c

```diff
@@ -1281,6 +1281,9 @@ SetFormatAndEncodings(rfbClient* client)
   se->pad = 0;
   se->nEncodings = 0;
 
+  if (client->appData.clipboardCap) /* add encoding for extendedclipboard */
```

Comment:
I think this can move to below the rfbEncodingQemuExtendedKeyEvent  stuff in line 1447 and without the clipboardCap config flag.

Reply:
clipboradcap is needed because these pseudo it is named after pseudo encoding, but its pattern is not a peseudo encoding...place here to make sure that even there is no pseudo encoding, clipboard is still announced.

Reply:
flag ok, but please move it down

Reply:
ok. I move it.